x/budget/spec/03_state_transitions.md

```diff
@@ -3,3 +3,9 @@
 # State Transitions
 
 This document describes the state transaction operations pertaining to the budget module.
+
+- TotalCollectedCoins
+
+## TotalCollectedCoins
+
+`TotalCollectedCoins` state transition occurs at each `BeginBlock`. You can read more about it in [04_begin_block.md](04_begin_block.md).
```

Comment:
```suggestion

- BudgetCollection
- Accumulate `TotalCollectedCoins`

## BudgetCollection

Get all budgets registered in `params.Budgets` and select the valid budgets to collect budgets for the block by its respective rate.
`BudgetCollection` state transition occurs at each `BeginBlock`. You can read more about it in [04_begin_block.md](04_begin_block.md).

## Cumulate TotalCollectedCoins

`TotalCollectedCoins` are cumulative coins collected in the budget since the bucket was created.
This state transition also occurs at each `BeginBlock` with `BudgetCollection`. You can read more about it in [04_begin_block.md](04_begin_block.md).
```

Reply:
Do you mean `## Cumulate TotalCollectedCoins` -> `## Accumulate TotalCollectedCoins`? in line 15?

Reply:
Oh, I made a mistake. Which one do you think is better?